newtab.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="newtab.css">
+    <title>Posh Tab</title>
+</head>
+
+<body>
+    <i class="fab fa-facebook-square" style="font-size:60px;color:blue;"></i>
+</body>
```

Comment:
That's the right syntax for a font awesome icon, but you also need to import it in your `<head>` tag. You can copy the tag from their website

Reply:
I did import the stylesheet in the <head> tag. Please check again.

Reply:
I'm not sure if that still works. I think the way to include FA has changed a bit:

```html
  <script src="https://kit.fontawesome.com/your_kit_number.js" crossorigin="anonymous"></script>
```

You get your kit number from their website after signing up

Reply:
I'll check once more. Thanks for the helpful suggestions.